src/main/cli/settings.c

```diff
@@ -469,10 +469,18 @@ static const char * const lookupTablePositionAltSource[] = {
     "DEFAULT", "BARO_ONLY", "GPS_ONLY"
 };
 
+<<<<<<< HEAD
 static const char * const lookupTableDshotBitbang[] = {
     "OFF", "ON", "AUTO"
 };
 
+=======
+>>>>>>> c1b13b139... various ff2_0 improvements
```

Comment:
should be rebased

Reply:
Well spotted, thanks. My bad, I force pushed after a failed rebase.